pilot/pkg/proxy/envoy/v1/config.go

```diff
@@ -35,6 +35,12 @@ import (
 	"istio.io/istio/pkg/log"
 )
 
+var (
+	// validateClustersDefault is an environment variable that can be set to false to disable
```

Comment:
Comment should start uppercase (how come the linters don't complain?)

I propose you name it "ValidateClusters" drop the misleading "Default"

Reply:
Relevant section of [Effective Go](https://golang.org/doc/effective_go.html#commentary):

> If every doc comment begins with the name of the item it describes, the output of godoc can usefully be run through grep. Imagine you couldn't remember the name "Compile" but were looking for the parsing function for regular expressions, so you ran the command,

Reply:
Yup, linter got it.